src/calligraph/geo.py

```diff
@@ -9,8 +11,8 @@
 from calligraph.core import filter_selectors
 
 # Transform from Web Mercator to Lat/Lon
-MERCATOR_TO_LATLON = Transformer.from_crs("EPSG:3857", "EPSG:4326")
-LATLON_TO_MERCATOR = Transformer.from_crs("EPSG:4326", "EPSG:3857")
+MERCATOR_TO_LATLON = Transformer.from_crs("EPSG:3857", "EPSG:4326", always_xy=True)
+LONLAT_TO_MERCATOR = Transformer.from_crs("EPSG:4326", "EPSG:3857", always_xy=True)
```

Comment:
Why this change from LATLON to LONLAT? Is there an issue with the code as it stands?

Reply:
LON -> x and LAT -> y, so you convert (lon, lat) to (x,y)and vice versa. Should be `MERCATOR_TO_LONLAT` too, actually.

This was causing an issue because it was doing (lat, lon) to (x, y) which geopandas/proj was getting confused by. I'll need to remind myself exactly how, it was 2 months ago. Should have left a comment ;)